### PR TITLE
Updated README to include codespaces details

### PR DESCRIPTION
**Pull request by sdarshale**
Add feedback link in readme file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,4 +45,7 @@ You'll need a version of node >= 16 which can be downloaded from the [Node.js we
 | Application | Codespaces Link |
 | ---- | ---- |
 | Advanced Integration | [![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/paypal-examples/docs-examples?devcontainer_path=.devcontainer%2Fadvanced-integration%2Fdevcontainer.json)|
-| Standard Integration | [![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/paypal-examples/docs-examples?devcontainer_path=.devcontainer%2Fstandard-integration%2Fdevcontainer.json)|
\ No newline at end of file
+| Standard Integration | [![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/paypal-examples/docs-examples?devcontainer_path=.devcontainer%2Fstandard-integration%2Fdevcontainer.json)|
+
+To report a bug or suggest a new feature, create an [issue in GitHub](https://github.com/paypal-examples/paypaldevsupport/issues/new/choose). \
```
Comment (gregjopa) [inline README.md:50]: Is this paypaldevsupport repo going to be public? Let's only reference it if its public.

Comment (sdarshale) [reply README.md:50]: Yes, the plan is to make it available publicly.

**Review by gregjopa — CHANGES_REQUESTED** [review]
Please hold off on merging. I want to continue the strategy discussion internally.

**Review by gregjopa — APPROVED** [review]

**Comment by MALIKSHAHID786** [comment]
Hi friend,s who r you

**Pull request merged**